packages/chains/src/polygon/redemptionStrategies.ts

```diff
@@ -153,6 +153,7 @@ const redemptionStrategies: { [token: string]: [RedemptionStrategyContract, stri
   ],
   [underlying(assets, assetSymbols["MAI-USDC"])]: [RedemptionStrategyContract.UniswapLpTokenLiquidator, USDC],
   [underlying(assets, assetSymbols["WMATIC-MATICx"])]: [RedemptionStrategyContract.UniswapLpTokenLiquidator, WMATIC],
+  [underlying(assets, assetSymbols.MIMO_PAR_80_20)]: [RedemptionStrategyContract.BalancerLpTokenLiquidator, "1"],
```

Comment:
this should not be `"1"` - it should be the the asset symbol of the output token - e.g. `assetSymbols.PAR`

Reply:
Actually it should be an underlying address, not a symbol

Reply:
@vminkov review here: using PAR as output, how would we get USDC back instead?